tests/cpu_bugs/spectre_v1.pm

```diff
@@ -0,0 +1,53 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: CPU BUGS on Linux kernel check
+# Maintainer: James Wang <jnwang@suse.com>
+
+package spectre_v1;
+use strict;
+use warnings;
+use base "consoletest";
+use base "Mitigation";
```

Comment:
Generally it is not recommended to use multiple inherit. What's the base class of Mitigation? Is it feasible to make inherit order like baseclass -> consoletest -> Mitigation -> spectre_v1?

Reply:
I could remove 'consoletest'